README.md

```diff
@@ -29,6 +29,36 @@ The Accessibility for Teams runs on Jekyll. To run it locally:
 - Run the web server with `jekyll serve`
 - Visit the local site at http://localhost:4000
 
+
+### Updating USWDS to a later version.
```

Comment:
These instructions help describe how to use the gulpfile. This also makes it somewhat easy to update USWDS versions in the future.